ManiVault/src/plugins/TextData/src/OrderedMap.cpp

```diff
@@ -128,6 +128,7 @@ namespace mv
         catch (std::out_of_range& e)
         {
             qWarning() << "Failed to find column: " << columnName;
+            qWarning() << "Caught exception: " << e.what();
```

Comment:
Is this necessary? The out of range exception gets called if it wasn't able to find the key in the map. I don't think more information is helpful. Perhaps slightly more elaboration in the original message.

Reply:
No, I mainly wanted to get rid of the warning. Marking it as [[maybe_unused]] gets the job done as well.

Reply:
Ah I see, should be able to remove the variable name as well right? just catch (std::out_of_range) or something

Reply:
But both are fine